Library/Homebrew/cmd/update.sh

```diff
@@ -321,6 +341,11 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
+  if [[ -n "$HOMEBREW_VERBOSE" ]]
+  then
+    oh1 "Fetching updates for all Git repositories"
```

Comment:
Feels like this is unnecessary given it'll be printed for each of the `$DIR` below?